README.md

```diff
@@ -1 +1,58 @@
-# js-leaderboard
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet) ![](https://img.shields.io/badge/JavaScript-yellow)
+
+# JavaScript Leaderboard
+
+This project is a JavaScript web application that acts as a leaderboard which allows users to:
+
+- Add their name and score to the leaderboard.
+- Reset the leaderboard.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Live Demo
+
+TBA
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps:
+
+### Install
+
+You can download the code to your personal computer by cloning the repository using the following command:
+
+```
+git clone
+```
+### Usage
+
+In order to run this project, enter the following command in your terminal:
+
+```
+npm start
+```
```

Comment:
Provide commands like ` npm install ` and `npm run dev`.

Reply:
This issue is resolved. Well done! :+1: